### PR TITLE
Expect protoc 3.9.2

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

**Review by helena — APPROVED** [review]
LGTM

File: project/Protobuf.scala [hunk 1]
```diff
@@ -32,7 +32,7 @@ object Protobuf {
     Compile / unmanagedJars ++= Seq(
         baseDirectory.value / ".." / "akka-protobuf-v3" / "target" / s"akka-protobuf-v3-assembly-${version.value}.jar"),
     protoc := "protoc",
-    protocVersion := "3.9.0",
+    protocVersion := "3.9.2",
```
Comment (patriknw) [inline project/Protobuf.scala]: do have to keep this in sync with the `protobuf-java` dependency? can we define it at one place?

Comment (chbatey) [reply project/Protobuf.scala]: Not for minor versions but let's do that anyway

Comment (patriknw) [reply project/Protobuf.scala]: will you change to a shared `val` or do we leave it as this?

**Pull request closed without merging**